letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -761,7 +798,16 @@ elif [ -f /etc/redhat-release ]; then
   if [ "$RPM_DIST_NAME" = "fedora" ]; then
     RPM_DIST_VERSION=`(. /etc/os-release 2> /dev/null && echo $VERSION_ID) || echo "0"`
   fi
-  if [ "$RPM_DIST_NAME" = "fedora" -a "$RPM_DIST_VERSION" -ge 29 -o "$PYVER" -eq 26 ]; then
+  if [ "$PYVER" -eq 26 ]; then
+    Bootstrap() {
+      BootstrapMessage "Legacy RedHat-based OSes that will use Python3"
+      BootstrapRpmPython3Legacy
+    }
+    USE_PYTHON_3=1
+    BOOTSTRAP_VERSION="BootstrapRpmPython3Legacy $BOOTSTRAP_RPM_PYTHON3_LEGACY_VERSION"
+    # Insert the SCL specific path in PATH to resolve the correct python binaries from SCL
+    PATH="/opt/rh/rh-python36/root/usr/bin:$PATH"
+  elif [ "$RPM_DIST_NAME" = "fedora" -a "$RPM_DIST_VERSION" -ge 29 -o "$PYVER" -eq 26 ]; then
     Bootstrap() {
       BootstrapMessage "RedHat-based OSes that will use Python3"
```

Comment:
I can see here that the "RedHat" spelling was present even before this PR.

Reply:
I take the occasion here to fix that.